backend/watermark/helpers.py

```diff
@@ -0,0 +1,79 @@
+
+from backend.watermark.extended_watermark_processor import WatermarkLogitsProcessor
+from backend.watermark.extended_watermark_processor import WatermarkDetector
+
+from transformers import (AutoTokenizer,
+                          AutoModelForSeq2SeqLM,
+                          AutoModelForCausalLM,
+                          LogitsProcessorList)
+import torch
+import argparse
+from argparse import Namespace
+
+arg_dict = {
+    'run_gradio': True, 
+    'demo_public': False, 
+     'model_name_or_path': 'facebook/opt-125m', 
```

Comment:
if it is possible can we test with large models? @muratbayrktr

Reply:
I believe we can, what is the size of the `large models` you refer?

Reply:
#'model_name_or_path': 'facebook/opt-6.7b',
    # 'model_name_or_path': 'facebook/opt-13b',